packages/google_maps_flutter/google_maps_flutter/example/ios/Podfile

```diff
@@ -28,6 +28,9 @@ require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelpe
 flutter_ios_podfile_setup
 
 target 'Runner' do
+  use_frameworks!
+  use_modular_headers!
```

Comment:
`Google-Maps-iOS-Utils` does not work without these. However, these lines are included in newer Podfiles generated by Flutter.

Without these lines, the iOS app crashes with the following error:
<img width="1112" alt="Screen Shot 2022-07-14 at 6 01 05 PM" src="https://user-images.githubusercontent.com/7896519/179096009-84c6fb6a-3dd1-4f27-893d-3421937e7610.png">

Reply:
Yep, thats essentially what I was asking. So this change is good! thanks for clarifying.

Reply:
`use_modular_headers!` is not added for Objective-C projects because it should not be necessary.  We should not be adding this in the example app, that implies all consumers of this plugin will also need to manually edit this file.  

Let's understand what's going on here.  Why is this needed?  Is the utils pod malformed (I'm guessing they didn't implement modules correctly)? Can we upstream a fix? Can we need to work around it the flutter podspec?

Reply:
Maybe only `use_frameworks!` is necessary? I only added both because they're both in the default Podfile generated by flutter.

Reply:
Removed `use_modular_headers!` and the example app still runs

Reply:
Flutter Objective-C projects do not have `use_frameworks!` in the Podfile, they use libraries, not frameworks.  Just updating it in this example project makes the tests pass, but that's not how actual Flutter apps are set up.  As-is this PR will crash all real Flutter Objective-C projects on launch.  We need to get this working with _only_ changes in the plugin and podspec, not by changing things in the example project.

The problem is that `Google-Maps-iOS-Utils` is a static framework, but also contains Swift files.  It's being statically linked into the app, but the app target isn't set up to link on the Swift runtime because it has no Swift classes itself.
https://github.com/googlemaps/google-maps-ios-utils/blob/39349fc28a0bcf8cfe817e4758be42cb30be710c/Google-Maps-iOS-Utils.podspec#L25

That's also why the pod linter failed when you were missing `swift_version` in the `google_maps_flutter_ios` podspec even though there's no Swift code in that plugin.
https://github.com/flutter/plugins/pull/5274#discussion_r921704285

Please revert all changes to the `Podfile`.  I believe this will work without changes to the example app if you add this to the podspec (note `swift_version` should be 5, not 4):
```ruby

  # "Google-Maps-iOS-Utils" is static and contains Swift classes.
  # Find the Swift runtime when these plugins are built as libraries without `use_frameworks!`
  s.swift_version = '5.0'
  s.xcconfig = {
    'LIBRARY_SEARCH_PATHS' => '$(inherited) $(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)/ $(SDKROOT)/usr/lib/swift',
    'LD_RUNPATH_SEARCH_PATHS' => '$(inherited) /usr/lib/swift',
  }
```

Reply:
Done